diff --git a/blocks/commerce-account-header/commerce-account-header.js b/blocks/commerce-account-header/commerce-account-header.js
index 2ccd5c68..c0f04921 100644
--- a/blocks/commerce-account-header/commerce-account-header.js
+++ b/blocks/commerce-account-header/commerce-account-header.js
@@ -3,7 +3,7 @@ import { readBlockConfig } from '../../scripts/aem.js';
 
 export default function decorate(block) {
   const {
-    title = 'My Account',
+    title = 'My account',
   } = readBlockConfig(block);
 
   block.innerHTML = '';
