diff --git a/blocks/enrichment/enrichment.js b/blocks/enrichment/enrichment.js
index 2c76e0b6..3b6801f4 100644
--- a/blocks/enrichment/enrichment.js
+++ b/blocks/enrichment/enrichment.js
@@ -1,5 +1,5 @@
 import { readBlockConfig } from '../../scripts/aem.js';
-import { getProductSku, fetchIndex } from '../../scripts/commerce.js';
+import { getProductSku, fetchIndex, IS_UE } from '../../scripts/commerce.js';
 import { loadFragment } from '../fragment/fragment.js';
 
 export default async function decorate(block) {
@@ -7,6 +7,10 @@ export default async function decorate(block) {
 
   try {
     const filters = {};
+    if (!type) {
+      throw new Error('No type found in enrichment block configuration');
+    }
+
     if (type === 'product') {
       const productSku = getProductSku();
       if (!productSku) {
@@ -16,12 +20,13 @@ export default async function decorate(block) {
     }
 
     if (type === 'category') {
-      const plpBlock = document.querySelector('.block.product-list-page');
+      // Look for PLP block using "product-list-page" block selector
+      const plpBlock = document.querySelector('.product-list-page');
       if (!plpBlock) {
         throw new Error('No product list page block found');
       }
 
-      const category = plpBlock.dataset?.category || readBlockConfig(plpBlock).category;
+      const category = plpBlock.dataset?.urlpath || readBlockConfig(plpBlock).urlpath;
       if (!category) {
         throw new Error('No category ID found in product list page block');
       }
@@ -35,7 +40,7 @@ export default async function decorate(block) {
     const index = await fetchIndex('enrichment/enrichment');
     const matchingFragments = index.data
       .filter((fragment) => Object.keys(filters).every((filterKey) => {
-        const values = fragment[filterKey];
+        const values = JSON.parse(fragment[filterKey]);
         return values.includes(filters[filterKey]);
       }))
       .map((fragment) => fragment.path);
@@ -63,6 +68,7 @@ export default async function decorate(block) {
   } catch (error) {
     console.error(error);
   } finally {
-    block.closest('.enrichment-wrapper')?.remove();
+    // don't remove wrapper in UE because then it will not be authorable
+    if (!IS_UE) block.closest('.enrichment-wrapper')?.remove();
   }
 }
