diff --git a/blocks/product-recommendations/product-recommendations.js b/blocks/product-recommendations/product-recommendations.js
index 29b0bed1..0ca3d024 100644
--- a/blocks/product-recommendations/product-recommendations.js
+++ b/blocks/product-recommendations/product-recommendations.js
@@ -17,7 +17,6 @@ import { publishRecsItemAddToCartClick } from '@dropins/storefront-recommendatio
 // Wishlist Dropin
 import { WishlistToggle } from '@dropins/storefront-wishlist/containers/WishlistToggle.js';
 import { render as wishlistRender } from '@dropins/storefront-wishlist/render.js';
-import { HeartNormal, HeartAdd } from '../../scripts/wishlist-icons.js';
 
 // Block-level
 import { readBlockConfig } from '../../scripts/aem.js';
@@ -117,9 +116,7 @@ export default async function decorate(block) {
       container.innerHTML = '';
     }
 
-    const storeViewCode = getConfigValue('headers.cs.Magento-Store-View-Code')
-      || getConfigValue('headers.cs.AC-View-ID')
-      || 'default';
+    const storeViewCode = getConfigValue('headers.cs.Magento-Store-View-Code');
     const createProductLink = (item) => getProductLink(item.urlKey, item.sku);
 
     // Get product view history
@@ -205,8 +202,6 @@ export default async function decorate(block) {
               // Render Icon
               wishlistRender.render(WishlistToggle, {
                 product: ctx.item,
-                iconToWishlist: HeartNormal,
-                iconWishlisted: HeartAdd,
               })($wishlistToggle);
 
               // Append to Cart Item
@@ -323,7 +318,5 @@ export default async function decorate(block) {
       });
     });
     inViewObserver.observe(section);
-  } else {
-    debouncedLoadRecommendation(false);
   }
 }
