diff --git a/scripts/__dropins__/storefront-order/fragments.js b/scripts/__dropins__/storefront-order/fragments.js
index c9c5d579..e3bd8f17 100644
--- a/scripts/__dropins__/storefront-order/fragments.js
+++ b/scripts/__dropins__/storefront-order/fragments.js
@@ -28,6 +28,7 @@ const d = `fragment REQUEST_RETURN_ORDER_FRAGMENT on Return {
   name
   sku
   only_x_left_in_stock
+  gift_wrapping_available
   gift_wrapping_price {
     currency
     value
