diff --git a/scripts/__dropins__/storefront-order/fragments.original.js b/scripts/__dropins__/storefront-order/fragments.original.js
index 126c4b27..2e10c32f 100644
--- a/scripts/__dropins__/storefront-order/fragments.original.js
+++ b/scripts/__dropins__/storefront-order/fragments.original.js
@@ -35,6 +35,7 @@ const d=`
     name
     sku
     only_x_left_in_stock
+    gift_wrapping_available
     gift_wrapping_price {
       currency
       value
