diff --git a/scripts/__dropins__/storefront-checkout/containers/PaymentMethods/handlers.d.ts b/scripts/__dropins__/storefront-checkout/containers/PaymentMethods/handlers.d.ts
index c5c8496f..551917d7 100644
--- a/scripts/__dropins__/storefront-checkout/containers/PaymentMethods/handlers.d.ts
+++ b/scripts/__dropins__/storefront-checkout/containers/PaymentMethods/handlers.d.ts
@@ -4,7 +4,10 @@ export declare enum HandlerCode {
     PaymentOnAccount = "companycredit",
     PurchaseOrder = "purchaseorder"
 }
-export declare const handleRefNumberChange: (code: string, isRequired: boolean) => (this: any, ...args: any[]) => void;
+export declare const handleRefNumberChange: (code: string, isRequired: boolean) => {
+    (this: any, ...args: any[]): void;
+    cancel(): void;
+};
 export declare const resetHandlersCache: () => void;
 export declare const createHandler: (code: HandlerCode) => PaymentMethodConfig;
 export declare const defaultHandlers: {
