diff --git a/scripts/__dropins__/storefront-order/lib/categorizeProducts.d.ts b/scripts/__dropins__/storefront-order/lib/categorizeProducts.d.ts
index 970ba4e7..cf305007 100644
--- a/scripts/__dropins__/storefront-order/lib/categorizeProducts.d.ts
+++ b/scripts/__dropins__/storefront-order/lib/categorizeProducts.d.ts
@@ -8,6 +8,7 @@ export declare const categorizeProducts: (order: OrderDataModel) => {
             recipientName: string;
             message: string;
         };
+        giftWrappingAvailable: boolean;
         giftWrappingPrice: import('../types').MoneyProps;
         productGiftWrapping: {
             uid: string;
