diff --git a/scripts/__dropins__/storefront-wishlist/chunks/removeProductsFromWishlist.js b/scripts/__dropins__/storefront-wishlist/chunks/removeProductsFromWishlist.js
index 69027a7b..acafa4fa 100644
--- a/scripts/__dropins__/storefront-wishlist/chunks/removeProductsFromWishlist.js
+++ b/scripts/__dropins__/storefront-wishlist/chunks/removeProductsFromWishlist.js
@@ -15,5 +15,5 @@ import{events as d}from"@dropins/tools/event-bus.js";import{FetchGraphQL as S}fr
       }
     }
   }
-`,x=async s=>{var r,o,i;const e=p(),t={...e,items:(r=e.items)==null?void 0:r.filter(h=>!s.some(l=>{var a;return u(h,{sku:l.product.sku,optionUIDs:(a=l.selectedOptions)==null?void 0:a.map(I=>I.uid)})}))};if(n.authenticated){if(!n.wishlistId)throw Error("Wishlist ID is not set");const h=s.map(m=>m.id),{errors:l,data:a}=await w(O,{variables:{wishlistId:n.wishlistId,wishlistItemsIds:h}}),I=[...((o=a==null?void 0:a.removeProductsFromWishlist)==null?void 0:o.user_errors)??[],...l??[]];if(I.length>0)return d.emit("wishlist/data",e),D(I)}return t.items_count=(i=t.items)==null?void 0:i.length,d.emit("wishlist/data",t),null};export{P as a,F as b,R as c,M as d,N as e,w as f,p as g,D as h,u as i,E as j,H as k,L as l,x as r,n as s};
+`,x=async s=>{var r,o,i;const e=p(),t={...e,items:(r=e.items)==null?void 0:r.filter(h=>!s.some(l=>{var a;return u(h,{sku:l.product.sku,optionUIDs:(a=l.selectedOptions)==null?void 0:a.map(I=>I.uid)})}))};if(n.authenticated){if(!n.wishlistId)throw Error("Wishlist ID is not set");const h=s.map(m=>m.id),{errors:l,data:a}=await w(O,{variables:{wishlistId:n.wishlistId,wishlistItemsIds:h}}),I=[...((o=a==null?void 0:a.removeProductsFromWishlist)==null?void 0:o.user_errors)??[],...l??[]];if(I.length>0)return d.emit("wishlist/data",e),D(I)}return t.items_count=(i=t.items)==null?void 0:i.length,d.emit("wishlist/data",t),null};export{N as a,L as b,H as c,R as d,P as e,w as f,p as g,D as h,u as i,F as j,M as k,E as l,x as r,n as s};
 //# sourceMappingURL=removeProductsFromWishlist.js.map
