diff --git a/blocks/commerce-cart/commerce-cart.js b/blocks/commerce-cart/commerce-cart.js
index 133389b5..26d45ab0 100644
--- a/blocks/commerce-cart/commerce-cart.js
+++ b/blocks/commerce-cart/commerce-cart.js
@@ -20,9 +20,8 @@ import { render as wishlistRender } from '@dropins/storefront-wishlist/render.js
 import { WishlistToggle } from '@dropins/storefront-wishlist/containers/WishlistToggle.js';
 import { WishlistAlert } from '@dropins/storefront-wishlist/containers/WishlistAlert.js';
 import { tryRenderAemAssetsImage } from '@dropins/tools/lib/aem/assets.js';
-
-// API
 import { publishShoppingCartViewEvent } from '@dropins/storefront-cart/api.js';
+import { HeartNormal, HeartAdd } from '../../scripts/wishlist-icons.js';
 
 // Modal and Mini PDP
 import createMiniPDP from '../../scripts/components/commerce-mini-pdp/commerce-mini-pdp.js';
@@ -33,7 +32,9 @@ import '../../scripts/initializers/cart.js';
 import '../../scripts/initializers/wishlist.js';
 
 import { readBlockConfig } from '../../scripts/aem.js';
-import { fetchPlaceholders, rootLink, getProductLink } from '../../scripts/commerce.js';
+import {
+  fetchPlaceholders, rootLink, getProductLink, checkIsAuthenticated,
+} from '../../scripts/commerce.js';
 
 export default async function decorate(block) {
   // Configuration
@@ -217,19 +218,23 @@ export default async function decorate(block) {
             ctx.appendChild(editLink);
           }
 
-          // Wishlist Button (if product is not configurable)
-          const $wishlistToggle = document.createElement('div');
-          $wishlistToggle.classList.add('cart__action--wishlist-toggle');
-
-          wishlistRender.render(WishlistToggle, {
-            product: ctx.item,
-            size: 'medium',
-            labelToWishlist: placeholders?.Global?.CartMoveToWishlist,
-            labelWishlisted: placeholders?.Global?.CartRemoveFromWishlist,
-            removeProdFromCart: Cart.updateProductsFromCart,
-          })($wishlistToggle);
+          // Wishlist Button (only for logged-in users)
+          if (checkIsAuthenticated()) {
+            const $wishlistToggle = document.createElement('div');
+            $wishlistToggle.classList.add('cart__action--wishlist-toggle');
 
-          ctx.appendChild($wishlistToggle);
+            wishlistRender.render(WishlistToggle, {
+              product: ctx.item,
+              size: 'medium',
+              iconToWishlist: HeartAdd,
+              iconWishlisted: HeartNormal,
+              labelToWishlist: placeholders?.Global?.CartMoveToWishlist,
+              labelWishlisted: placeholders?.Global?.CartRemoveFromWishlist,
+              removeProdFromCart: Cart.updateProductsFromCart,
+            })($wishlistToggle);
+
+            ctx.appendChild($wishlistToggle);
+          }
 
           // Gift Options
           const giftOptions = document.createElement('div');
