diff --git a/scripts/__dropins__/storefront-order/fragments.original.js b/scripts/__dropins__/storefront-order/fragments.original.js
index 8cd982af..126c4b27 100644
--- a/scripts/__dropins__/storefront-order/fragments.original.js
+++ b/scripts/__dropins__/storefront-order/fragments.original.js
@@ -1,6 +1,6 @@
-/*! Copyright 2025 Adobe
+/*! Copyright 2026 Adobe
 All Rights Reserved. */
-const s=`
+const d=`
   fragment REQUEST_RETURN_ORDER_FRAGMENT on Return {
     __typename
     uid
@@ -280,14 +280,14 @@ const s=`
       currency
     }
   }
-`,o=`
+`,A=`
   fragment GIFT_MESSAGE_FRAGMENT on GiftMessage {
     __typename
     from
     to
     message
   }
-`,A=`
+`,o=`
   fragment GIFT_WRAPPING_FRAGMENT on GiftWrapping {
     __typename
     uid
@@ -386,15 +386,15 @@ const s=`
   ${u}
   ${E}
   ${n}
-  ${o}
   ${A}
+  ${o}
   ${a}
   ${i}
   ${c}
   ${t}
   ${r}
   ${T}
-`,d=`
+`,s=`
   fragment PLACE_ORDER_FRAGMENT on PlaceOrderOutput {
     errors {
       code
@@ -407,17 +407,16 @@ const s=`
 
   ${e}
 `,l=`
-  fragment PLACE_NEGOTIABLE_QUOTE_ORDER_FRAGMENT on PlaceNegotiableQuoteOrderOutput {
-    # TODO: When errors prop is available in the response, use it to handle errors
-    # errors {
-    #   code
-    #   message
-    # }
-    orderV2 {
+  fragment PLACE_NEGOTIABLE_QUOTE_ORDER_FRAGMENT on PlaceNegotiableQuoteOrderOutputV2 {
+    errors {
+      code
+      message
+    }
+    order {
       ...GUEST_ORDER_FRAGMENT
     }
   }
 
   ${e}
-`;export{_ as ADDRESS_FRAGMENT,u as APPLIED_GIFT_CARDS_FRAGMENT,E as BUNDLE_ORDER_ITEM_DETAILS_FRAGMENT,R as DOWNLOADABLE_ORDER_ITEMS_FRAGMENT,n as GIFT_CARD_DETAILS_FRAGMENT,o as GIFT_MESSAGE_FRAGMENT,A as GIFT_WRAPPING_FRAGMENT,e as GUEST_ORDER_FRAGMENT,a as ORDER_ITEM_DETAILS_FRAGMENT,i as ORDER_ITEM_FRAGMENT,c as ORDER_SUMMARY_FRAGMENT,l as PLACE_NEGOTIABLE_QUOTE_ORDER_FRAGMENT,d as PLACE_ORDER_FRAGMENT,t as PRICE_DETAILS_FRAGMENT,r as PRODUCT_DETAILS_FRAGMENT,s as REQUEST_RETURN_ORDER_FRAGMENT,T as RETURNS_FRAGMENT};
+`;export{_ as ADDRESS_FRAGMENT,u as APPLIED_GIFT_CARDS_FRAGMENT,E as BUNDLE_ORDER_ITEM_DETAILS_FRAGMENT,R as DOWNLOADABLE_ORDER_ITEMS_FRAGMENT,n as GIFT_CARD_DETAILS_FRAGMENT,A as GIFT_MESSAGE_FRAGMENT,o as GIFT_WRAPPING_FRAGMENT,e as GUEST_ORDER_FRAGMENT,a as ORDER_ITEM_DETAILS_FRAGMENT,i as ORDER_ITEM_FRAGMENT,c as ORDER_SUMMARY_FRAGMENT,l as PLACE_NEGOTIABLE_QUOTE_ORDER_FRAGMENT,s as PLACE_ORDER_FRAGMENT,t as PRICE_DETAILS_FRAGMENT,r as PRODUCT_DETAILS_FRAGMENT,d as REQUEST_RETURN_ORDER_FRAGMENT,T as RETURNS_FRAGMENT};
 //# sourceMappingURL=fragments.js.map
