diff --git a/scripts/__dropins__/storefront-pdp/i18n/en_US.json.d.ts b/scripts/__dropins__/storefront-pdp/i18n/en_US.json.d.ts
index e94ee410..01e8ccbc 100644
--- a/scripts/__dropins__/storefront-pdp/i18n/en_US.json.d.ts
+++ b/scripts/__dropins__/storefront-pdp/i18n/en_US.json.d.ts
@@ -19,6 +19,12 @@ declare const _default: {
       "SpecialPrice": {
         "label": "Special Price"
       },
+      "TierPrice": {
+        "label": "Tier Price"
+      },
+      "TierPricing": {
+        "label": "Buy {quantity} for {price} each and save {percent}%"
+      },
       "PriceRange": {
         "From": {
           "label": "From"
