diff --git a/scripts/__dropins__/storefront-pdp/components/index.d.ts b/scripts/__dropins__/storefront-pdp/components/index.d.ts
index f35b6880..213b7cf3 100644
--- a/scripts/__dropins__/storefront-pdp/components/index.d.ts
+++ b/scripts/__dropins__/storefront-pdp/components/index.d.ts
@@ -22,5 +22,6 @@ export * from './Swatches';
 export * from './Zoom';
 export * from './Header';
 export * from './Price';
+export * from './PriceTiers';
 export * from './GiftCardOptions';
 //# sourceMappingURL=index.d.ts.map
\ No newline at end of file
